packages/web-client/app/components/card-pay/layer-one-connect-modal/index.css

```diff
@@ -1,4 +1,4 @@
 .layer-one-connect-modal__card {
-  max-height: 48rem;
+  max-height: 22.8rem;
```

Comment:
@Aierie this is a fragile solution to the problem of sizing the modal height properly. Do you have any suggestions on how to tackle this?

Reply:
You could replace it with `height:auto`. This happens because of `height:100%` on `Boxel::CardContainer`.

Reply:
Would also add `top: 50%; transform: translateY(-50%);` for centering.